verification/certifyKey_test.go

```diff
@@ -14,19 +15,43 @@ import (
 	"github.com/zmap/zlint/v3/lint"
 )
 
-// This file is used to test the certify key command by using a simulator
+// This file is used to test the certify key command by using a simulator/emulator
+
+func GetTestTarget_CertifyKey() ([]TestDPEInstance, error) {
+
+	// Added dummy support for emulator
+	support_needed := []string{"AutoInit", "X509"}
+
+	var instances []TestDPEInstance
+
+	if testTargetType == EMULATOR {
+		return GetEmulatorTarget(support_needed, instances)
+
+	} else if testTargetType == SIMULATOR {
+		instances = []TestDPEInstance{
+			// No extra options besides AutoInit.
+			&DpeSimulator{exe_path: *socket_exe, supports: Support{AutoInit: true, X509: true}},
+			// Supports AutoInit and simulation contexts.
+			&DpeSimulator{exe_path: *socket_exe, supports: Support{AutoInit: true, Simulation: true, X509: true}},
```

Comment:
Instead of making multiple instances  with different support matrices, I think we should have a separate test for each different support configuration.

Reply:
Regarding the comment, my understanding is we need to create a seperate  test for each support instead of muliptle dpe instances with mulitple supports. I have added a snippet with my understanding. Shall I proceed with this.
	
	 For example:
		func TestCertifyKey_<name1>(t *testing.T){
			support_needed := []string{"AutoInit", "X509"}
			
			instance, err := GetTestTarget_CertifyKey(support_needed)
			if err != nil {
			...
			}

			testCertifyKey(instance, t)		
		}
		
		func TestCertifyKey_<name2>(t *testing.T){
			support_needed := []string{"AutoInit", "X509","Simulation"}
			
			instance, err := GetTestTarget_CertifyKey(support_needed)
			if err != nil {
			...
			}

			testCertifyKey(instance, t)		
		}
		
		-----------------------------------------------------------------
		
		func GetTestTarget_CertifyKey(support_needed []string) (TestDPEInstance, error) {
			if testTargetType == EMULATOR {
				//Create the dpe instance with the support_needed if it is a emulator
			} else if testTargetType == SIMULATOR {
				//Create the dpe instance with the support_needed if it is a simulator
			}
			return instance, nil
		}

Reply:
Addressed